docs/reference/commandline/dockerd.md

```diff
@@ -1147,6 +1147,7 @@ This is a full example of the allowed configuration options on Linux:
 	"ip-forward": false,
 	"ip-masq": false,
 	"userland-proxy": false,
+	"userland-proxy-path": "/usr/libexec/my-docker-proxy",
```

Comment:
I think it's ok to use this for illustration. The runtime path below also contains an example ("/usr/local/bin/my-runc-replacement")